feature-group-definitions/cap.dist.yml

```diff
@@ -0,0 +1,18 @@
+# Generated from: cap.yml
+# Do not edit this file by hand. Edit the source file instead!
+# Last modified: 2024-04-04T09:35:23.998323918+00:00[UTC]
+
+name: cap unit
+description: The CSS `cap` unit corresponds to the height of Latin capital letters.
+spec: https://drafts.csswg.org/css-values-4/#cap
```

Comment:
This file demonstrates the real dream. Only these lines were authored! The status and compat features came from `compute-baseline` and BCD.